akka-docs-dev/rst/java/stream-graphs.rst

```diff
@@ -194,6 +194,24 @@ together and also turned around with the ``.reversed()`` method. The test
 simulates both parties of a network communication protocol without actually
 having to open a network connection—the flows can just be connected directly.
 
+.. _graph-matvalue-java:
+
+Accessing the materialized value inside the Graph
+-------------------------------------------------
+
+In certain cases it might be necessary to feed back the materialized value of a Graph (partial, closed or backing a
+Source, Sink, Flow or BidiFlow). This is possible by using ``builder.matValue`` which gives an ``Outlet`` that
```

Comment:
It is just long. While here it might not matter that much, it starts to matter when you use the Flow DSL:

    SourceX.viaMapMaterialized(f1)(Keep.both).viaMapMaterialized(f2)(Keep.left).toMapMaterialized(s)(Keep.right)

compared to

    SourceX.viaMat(f1)(Keep.both).viaMat(f2)(Keep.left).toMat(s)(Keep.right)